### ISSUE TITLE
Feature request: control garage lights

### ISSUE THREAD

**Issue by danepowell** [OpenGarage/OpenGarage-Hardware#8]
Most garage doors include 120 V lights on the power head that automatically turn on when the garage opens, and can also be controlled by a switch on the opener.

It would be awesome to control these lights with OpenGarage. I'm guessing this would require a second relay, since the light button has its own control wire (green for old Genie models like mine).

**Comment by rayshobby** [comment]
I've looked at this a while back. This highly depends on the garage door system. I have a LiftMaster (non-security+ 2.0), and I noticed its door button has a light control switch. It uses the same two wires for door control and light control. I took apart the door button panel, and found that the door control simply shorts the two wires (as expected), and the light control connects a 1uF capacitor across the two wires. Some online posts also documented this feature:
https://forum.arduino.cc/t/using-esp8266-arduino-and-2n2222-for-garage-door/472101
I've actually made a variant of OpenGarage that uses two solid state relays, the first one simply shorts the two wires, and the second one has a 1uF capacitor in series. This allows for controlling both the door and the light.

However, if a garage door system uses security+ 2.0, then it's a completely different story -- it uses rolling code so is not as simple as connecting a 1uF capacitor across the two wires. I am sure other brands would use yet different ways for light control. So I concluded it would be hard to design one circuit that handles all these different ways for light control.
